core/src/main/java/tachyon/master/MasterClient.java

```diff
@@ -300,6 +300,35 @@ public synchronized long getUserId() throws IOException {
     return null;
   }
 
+  public synchronized long getUsedBytes() throws IOException {
```

Comment:
Could we just expose the MasterInfo's getUsedBytes and getCapacityBytes instead of redoing the logic here?